api/app.js

```diff
@@ -52,8 +53,10 @@ app.use(async (req, res, next) => {
         if (response.status === 200) {
           next()
         } else {
+          throw new Error('An authentication error occurred.')
         }
       } catch (err) {
+        console.log({ error: err, request: req })
```

Comment:
@mbwatson I think it'd be good to log the response here as well.

Reply:
@mbwatson Or perhaps it is not needed since the `err` will catch it. Yeah, now that I think about it, it must be the case. So please disregard my previous comment.

Reply:
@hyi yeah, the error contains the response, but it request, too. i thought it might be handy to bring it to the top level of the logged object. so we wouldn't actually lose any information with just a `console.log(err)`, it just might be slightly harder to find the request and response buried in there. what do you think?

Reply:
@mbwatson Yeah, I agree. It'd be a bit easier to find the request and response if logging them separately even if they are included in the error. But since they are included in the error, it is not critical to log them separately. So I am fine either way. Whatever you think will work better works for me.

Reply:
:+1: